src/App.tsx

```diff
@@ -1,20 +1,23 @@
+import { Navigate, Route, Routes } from 'react-router-dom';
 import { PeoplePage } from './components/PeoplePage';
-import { Navbar } from './components/Navbar';
-
+import { NavBar } from './components/NavBar';
```

Comment:
```suggestion
import { NavBar } from './components/Navbar';
```
<img width="556" alt="image" src="https://github.com/mate-academy/react_people-table-advanced/assets/70220667/cf89145d-c359-47a8-970c-7f1825196128">

Reply:
<img width="1480" alt="image" src="https://github.com/mate-academy/react_people-table-advanced/assets/70220667/b70c243c-f2a7-4446-bc4a-8dc188a950c0">
<img width="1329" alt="image" src="https://github.com/mate-academy/react_people-table-advanced/assets/70220667/aab5efbf-7048-4164-9759-c2276857d4be">
same here